admin/con-rhdh-integration-aws.adoc

```diff
@@ -0,0 +1,7 @@
+[id='con-rhdh-integration-aws_{context}']
+= {product} Integration with Amazon Web Services (AWS)
+
+You can integrate your {product} application with Amazon Web Services (AWS), which can help you streamline your workflows within the AWS ecosystem. Integrating the {product-short} resources with AWS provides access to a comprehensive suite of tools, services, and solutions.
+
+The integration with AWS requires the deployment of {product-short} in Elastic Kubernetes Services (EKS) using Helm Chart and Operator.
```

Comment:
```suggestion
The integration with AWS requires the deployment of {product-short} in Elastic Kubernetes Services (EKS) using Helm Chart or Operator.
```

This should an OR, right? We are using either the Helm Chart or the Operator.

Reply:
Done